crates/provider/src/heart.rs

```diff
@@ -108,18 +108,21 @@ impl<'a, N: Network, T: Transport + Clone> PendingTransactionBuilder<'a, N, T> {
     }
 
     /// Returns the number of confirmations to wait for.
```

Comment:
```suggestion
    /// Returns the number of required confirmations to wait for.
```
or is this requested?